core/src/main/scala/org/broadinstitute/dsde/rawls/workspace/WorkspaceService.scala

```diff
@@ -1551,7 +1557,7 @@ class WorkspaceService(protected val ctx: RawlsRequestContext,
 
   def sendChangeNotifications(workspaceName: WorkspaceName): Future[String] =
     for {
-      workspaceContext <- getWorkspaceContextAndPermissions(workspaceName, SamWorkspaceActions.own)
+      workspaceContext <- getV2WorkspaceContextAndPermissions(workspaceName, SamWorkspaceActions.own)
```

Comment:
Apparently there is no test coverage for this method. Should I add it?

Reply:
I think so.

Reply:
Added in https://github.com/broadinstitute/rawls/pull/2214/commits/215f0fd9af3b924d4f774c464e23262e55d0f283